### ISSUE TITLE
CWAG integ tests are currently red in about 90% of all cases

### ISSUE THREAD

**Issue by crasu** [magma/magma#13648]
Pipelined is currently not coming up cleanly on ci.

The log output looks like this:
```
Container restarting detected. Trying one more time
[vagrant@127.0.0.1:2222] run:  docker ps --format "***.Names***	***.Status***" | grep Restarting| grep --invert-match 'ocs2\|pcrf2'
[vagrant@127.0.0.1:2222] out: pipelined	Restarting (1) 1 second ago
[vagrant@127.0.0.1:2222] out: 
```

This behaviour started after removing the xwf containers from the build:
https://github.com/magma/magma/commit/e87b2c39

However as these containers are not used inthe integ tests it is unlikely that this issue is related to the xwf removal.

Sample run:
https://github.com/magma/magma/runs/7884346533?check_suite_focus=true

The test suite works in a local setup.

**Comment by crasu** [comment]
#13649

**Comment by sebathomas** [comment]
Fixed on master by #13649.
